src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/AuthenticationParameters/AzureAdProperties.cs

```diff
@@ -110,9 +112,7 @@ public AzureAdBlock(ApplicationParameters applicationParameters, JObject? existi
             TenantId = TenantId ?? DefaultProperties.TenantId,
             ClientId = ClientId ?? DefaultProperties.ClientId,
             CallbackPath = CallbackPath ?? DefaultProperties.CallbackPath,
-            Scopes = Scopes ?? DefaultProperties.ApiScopes,
-            ClientSecret = ClientSecret ?? DefaultProperties.ClientSecret,
```

Comment:
why do we not need these properties anymore? or if they are initialized elsewhere, why?

Reply:
Yeah they are only necessary when the app registration has a client secret (when a downstream API or MS graph is called), so I moved these properties to a separate object (DownstreamAiSettings)